db/migrate/20150122204353_create_password_resets.rb

```diff
@@ -0,0 +1,12 @@
+class CreatePasswordResets < ActiveRecord::Migration
+  def change
+    create_table :password_resets do |t|
+      t.integer :user_id, null: false
+      t.string :token, limit: 128, null: false
```

Comment:
I believe this limit was set for the confirmation_token previously. I don’t know what the rationale behind it was though. It’s also interesting because we’re limiting the token to a length of 20 bytes, so they don’t seem to match up. Thoughts on removing the limit?


On June 5, 2015 at 10:37:52 AM, Derek Prior (notifications@github.com) wrote:

In db/migrate/20150122204353_create_password_resets.rb:

> @@ -0,0 +1,12 @@
> +class CreatePasswordResets < ActiveRecord::Migration
> +  def change
> +    create_table :password_resets do |t|
> +      t.integer :user_id, null: false
> +      t.string :token, limit: 128, null: false
do you know what the significance of the limit is here?

—
Reply to this email directly or view it on GitHub.